crates/cargo-winrt/src/main.rs

```diff
@@ -326,24 +332,43 @@ fn try_download(url: String, recursion_amount: u8) -> anyhow::Result<Vec<u8>> {
                     let n = &header[9..12];
                     *status = Some(n.parse::<u16>().expect("Should be number"));
                 }
+                if header.starts_with("Content-Length: ") {
+                    *content_length = Some(header[16..header.len() - 2].to_owned())
+                }
                 true
             })
             .unwrap();
         debug!(" making request to {}", url);
         transfer.perform()?;
     }
+
+    let content_length = content_length.take().unwrap_or_default();
+    let cl: u64 = content_length.parse().unwrap_or_default();
```

Comment:
can you avoid ultra short names like `cl`? I'd rather just spell it out.

Reply:
Fixed.